package.json

```diff
@@ -144,7 +144,7 @@
   "scripts": {
     "docs": "node ./scripts/docs.js",
     "lint": "jshint lib/ system-test/ test/ && jscs lib/ system-test/ test/",
-    "test": "npm run docs && mocha test/*/*.js test/index.js test/docs.js",
+    "test": "npm run docs && mocha test/docs.js test/index.js test/*/*.js",
```

Comment:
I believe Mockery is not properly releasing a stub, or messing with the dependency tree altogether. Letting the docs run first is required to get around this.